src/deadline/blender_adaptor/BlenderClient/render_handlers/default_blender_handler.py

```diff
@@ -218,3 +219,35 @@ def set_scene_file(self, data: dict):
         if not os.path.isfile(file_path):
             raise FileNotFoundError(f"The scene file '{file_path}' does not exist")
         bpy.ops.wm.open_mainfile(filepath=file_path)
+
+    def set_render_device(self, data: dict) -> None:
+        """
+        Enables GPU rendering on the worker if specified in the user's configuration.
+
+        Args:
+            data (dict): The data given from the Adaptor. Keys expected: ['render_device']
+
+        Raises:
+            RuntimeError: If no devices of the specified type are available
+        """
+
+        if not data["render_device"] == "NONE":
```

Comment:
If someone is using an older submitter this will result in a `KeyError` and crash the adaptor. I'm not sure if that's the behaviour we want here. If the `render_device` key is missing should we print a warning and keep carrying on?

Reply:
That seems reasonable. I'm a little concerned about users expecting GPUs to be automatically enabled in earlier submitter versions, then wasting time rendering on CPUs, for which we _could_ fail the job instead...but at the same time, wasting time rendering on CPUs was the default behaviour beforehand, so a warning may be fine. Curious if others have thoughts!